### DISCUSSION TITLE
Physical meshes do not work for few mins after first render

### DISCUSSION THREAD

**Discussion by bhushan6** [pmndrs/use-cannon#246] [Q&A]
I have few 3d models and I am adding few cannon boxes on the positios of model for simple collision and it works for my project but only issue is that those boxes do not work for few mins after first render. Player passes through them for few initial mins but starts working after few mins properly. what could be the cause? I am using cannon's plane for a floor and its works properly bu this issue is only with those boxes

**Comment by drcmda** [comment]
impossible to say without a small repro. there are lots of examples where boxes are being used.
